src/LtiAssignmentsGradesService.php

```diff
@@ -41,7 +41,7 @@ public function putGrade(LtiGrade $grade, LtiLineitem $lineitem = null)
             $this->service_data['scope'],
             LtiServiceConnector::METHOD_POST,
             $score_url,
-            strval($grade),
+            $grade,
```

Comment:
What is the purpose of these changes?

Reply:
Guzzle expects a json object in the body. I was getting these errors because we were passing along a json blob converted into a string value.

<details>

![Screenshot from 2021-06-22 10-54-41](https://user-images.githubusercontent.com/8699045/122975653-6de8a000-d348-11eb-82e3-a697719ed9dc.png)


</details>